react-components/src/util/CopyTextArea.js

```diff
@@ -20,16 +20,17 @@ class CopyTextArea extends Component {
 
     onCopyText(e) {
         e.preventDefault();
-
-        this.refs.copyTextArea.select();
+        let ele = document.getElementById(`${this.props.debugIdPrefix}.CopyTextArea.TextField`);
+        ele.select();
         if (copySelection()) {
             this.setState({btnText: this.props.copiedText});
         }
     }
 
     componentDidMount() {
         if (!hasClipboardAPI()) {
-            this.refs.copyTextArea.select();
```

Comment:
Since we're getting rid of the `refs` usage here and above, we can also remove `ref="copyTextArea"` from the `TextField` in the `render` method below.

Reply:
👍